README.md

```diff
@@ -127,7 +127,7 @@ cd ostis-web-platform/scripts
   docker run -v ${PWD}:/workdir --rm -it ostis/scn-latex-plugin:latest "docs/main.tex"
   ```
 
-  After the compilation, the `main.pdf` file should appear at `ostis-web-platform/docs/`.
+  After the compilation, the `main.pdf` file should appear at `ostis-web-platform/docs/`. You can find more information about docs building [here](https://github.com/ostis-ai/scn-latex-plugin).
```

Comment:
```suggestion
  After the compilation, the `main.pdf` file should appear at `ostis-web-platform/docs/`. You can find more information about [scn-latex-plugin here](https://github.com/ostis-ai/scn-latex-plugin).
```
It should be "building the docs" rather than "docs building", but because the `scn-latex-plugin` is (theoretically) not only about the build process, it's better to rephrase it as "You can find more information about scn-latex-plugin here"